README.md

```diff
@@ -21,33 +21,34 @@ A simple Hello world page.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Have git correctly installed.
+
 ### Setup
 
+- Open a terminal window where you want to have installed a copy of the repository.
+
 ### Install
 
+- Use this command to get a copy:
+```
+git@github.com:helio3197/hello-microverse.git
```

Comment:
- [ ] This command is wrong, we use different commands to clone the repo. Kindly change this.